browser/css/jquery-ui-lightness.css

```diff
@@ -522,7 +522,6 @@ button.ui-button::-moz-focus-inner {
 .ui-dialog .ui-dialog-content {
 	position: relative;
 	border: 0;
-	padding: .5em 1em;
 	background: none;
```

Comment:
Please do not modify .ui-dialog .ui-dialog-content otherwise this is affecting many components including snackbars. Now it looks like:

![Screenshot_20230526_171133](https://github.com/CollaboraOnline/online/assets/65948705/dca7b59f-29bd-4138-a46c-558472418033)

better to change the padding in the tabs only